### ISSUE TITLE
Sheet Name Input Error

### ISSUE THREAD

**Issue by dgarretty** [mbleron/oracle#2]
I'm getting the following error when using the following code and xlsx file (i believe it's to do with the worksheet name input parameter)

[TEST.xlsx](https://github.com/mbleron/oracle/files/1130574/TEST.xlsx)

```
ORA-21560: argument 3 is null, invalid, or out of range
ORA-06512: at "SYS.DBMS_LOB", line 701
ORA-06512: at "OLAP_META_DATA.EXCELTABLE", line 303
ORA-06512: at "OLAP_META_DATA.EXCELTABLE", line 323
ORA-06512: at "OLAP_META_DATA.EXCELTABLE", line 1190
ORA-06512: at "OLAP_META_DATA.EXCELTABLE", line 1220
ORA-06512: at "OLAP_META_DATA.EXCELTABLE", line 1380
ORA-06512: at "OLAP_META_DATA.EXCELTABLEIMPL", line 61
ORA-06512: at line 1
```

```
select t.*
from table(
       ExcelTable.getRows(
         ExcelTable.getFile('DAT_DIR','TEST.xlsx')
       , 'DATA'
       , q'{ "COMPANY_CODE"       varchar2(10)  column 'A'
           , "PROFIT_CENTER"      varchar2(50)  column 'B'    
           , "LEVEL_NO"           varchar2(10)  column 'C'
           , "LEVEL1"             varchar2(100) column 'D'
           , "LEVEL2"             varchar2(100) column 'E'
           , "LEVEL3"             varchar2(100) column 'F'
           , "LEVEL4"             varchar2(100) column 'G'
           , "LEVEL5"             varchar2(100) column 'H'
           , "GL_CODE"            varchar2(20)  column 'I'
           , "FISCAL_YEAR"        varchar2(10)  column 'J'
           , "POSTING_PERIOD"     varchar2(10)  column 'K'
           , "PERIOD_ACTUAL"      varchar2(20)  column 'L'
           , "ACTUAL_YTD"         varchar2(20)  column 'M'  }'
       , 'A2'
       )
     ) t;

```

**Comment by mbleron** [comment]
Thanks for reporting the issue.

The workbook was not created by MS Excel, was it? Or if it was , I'm interested in knowing the exact version.

The problem comes from a feature of the ZIP format I don't handle, and which is not normally used in files created by Excel program.
In the local file header, when bit #3 of the General Purpose Bit Flag is set, each entry has an additional 12-byte data descriptor containing both compressed and uncompressed file sizes and the CRC-32.
In my current implementation, those sizes are retrieved as 0 instead of the actual values, hence the DBMS_LOB exception when passing an amount to read/copy equals to 0.

I'll get this fixed in the next few days.

**Comment by dgarretty** [comment]
No problem. The workbook was created by SAP Business Objects.

**Comment by mbleron** [comment]
Issue fixed in version 1.5
